### PR TITLE
Integrating DeltaLake

### PR DESCRIPTION
**Pull request by aamend**
Signed-off-by: antoine.amend <antoine.amend@databricks.com>

| Q                       | A                                |
| ----------------------- | -------------------------------- |
| Fixed issues?           |  |
| Tests added + passed?   | Yes                              |
| Changeset added?        | Yes                              |
| Documentation PR link   |                                  |
| Any dependency changes? | No                               |

### CODE REVIEW

**Comment by aamend** [comment]
I believe code is correct, but not that familiar with node applications and change sets. Appreciate if I could get some guidance getting this workflow successful

**Comment by akphi** [comment]
@aamend Hi, you can refer to https://github.com/finos/legend-studio/blob/master/CONTRIBUTING.md#changeset

File: .changeset/fair-beans-shout.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (epsstan) [inline .changeset/fair-beans-shout.md:1]: Please add a roundtrip test. See https://github.com/finos/legend-studio/pull/358 for an example.

**Comment by MauricioUyaguari** [comment]
@aamend looks good. Can we please add a grammar roundtrip test? 
See https://github.com/finos/legend-studio/tree/master/packages/legend-manual-tests/
Example of grammar cases: https://github.com/finos/legend-studio/tree/master/packages/legend-manual-tests/src/__tests__/roundtrip-grammar/cases
Should be able to add these now that the engine changes went in. 
Let me know if you have any questions. 
Thanks for your contribution!

File: .changeset/fair-beans-shout.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+'@finos/legend-studio': minor
```
Comment (akphi) [inline .changeset/fair-beans-shout.md:2]: Let's add `@finos/legend-graph: minor` to this one and remove it in the other changeset.

File: .changeset/chilled-ladybugs-push.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+'@finos/legend-graph': patch
+'@finos/legend-manual-tests': patch
+'@finos/legend-studio': patch
+---
+
+delta lake integration
```
Comment (akphi) [inline .changeset/chilled-ladybugs-push.md:7]: You can leave this one empty, practically it only contains change for `@finos/legend-manual-tests`

Comment (aamend) [reply .changeset/chilled-ladybugs-push.md:4]: done

File: .changeset/fair-beans-shout.md (hunk 2 above)
Comment (aamend) [reply .changeset/fair-beans-shout.md:2]: done

File: .changeset/chilled-ladybugs-push.md (hunk 3 above)
Comment (MauricioUyaguari) [reply .changeset/chilled-ladybugs-push.md:4]: @aamend  by empty, he might the message shoudl be empty but it should still have `@finos/legend-manual-tests`

Comment (MauricioUyaguari) [reply .changeset/chilled-ladybugs-push.md:4]: see https://raw.githubusercontent.com/finos/legend-studio/master/.changeset/weak-mugs-shake.md

Comment (MauricioUyaguari) [reply .changeset/chilled-ladybugs-push.md:4]: This causes the changeset build to fail

**Review by MauricioUyaguari — APPROVED** [review]

**Pull request merged**